backend/hct_mis_api/apps/core/fixtures.py

```diff
@@ -12,7 +12,7 @@
 def create_afghanistan(
     is_payment_plan_applicable: bool = False,
 ) -> BusinessArea:
-    return BusinessArea.objects.create(
+    return BusinessArea.objects.get_or_create(
```

Comment:
just curious... why this? @pkujawa

Reply:
When testing locally, since pytest, it happens that the Afghanistan is already there and tests throw the duplicate error